updates

pieckenst opened the pull request:
Замержь если хошь

В основном это бэкпорт из helia/canary тех вещей которые там есть - что называется почему бы и нет

pieckenst committed:
Update README.md

pieckenst committed:
Update README.md

pieckenst committed:
Add files via upload

pieckenst committed:
Update README.md

pieckenst committed:
backport

pieckenst committed:
localize fix

pieckenst committed:
Update config.json

pieckenst committed:
Буква и

pieckenst committed:
Update version.txt

pieckenst committed:
Локализация

pieckenst committed:
Merge branch 'master' of https://github.com/pieckenst/Openmod

arslee07 commented:
че там добавлено в основном

pieckenst commented:
> 
> 
> че там добавлено в основном

Музыка и пару команд 
wallpaper
kubik
monetka
casino
echo
embed
wiki (поиск с английской википедии)
Переделан about
добавлен shutdown
set_status
invite

pieckenst commented:
shutdown и set_status надо подредачить valid_users

pieckenst committed:
Теперь без кэша

arslee07 commented:
мб добавлю, но опенмод чисто для модерации

arslee07 commented:
казино, монетка :OB_trlo:

pieckenst commented:
> 
> 
> казино, монетка :OB_trlo:
собственно там ничего особенного

pieckenst commented:
> 
> 
> мб добавлю, но опенмод чисто для модерации

да и все равно особого плюса раздроблять ботов на отдельную функциональность не вижу - комбайн проще обслуживать чем следить за отдельными ботами для разного функционала

pieckenst committed:
коментарии

pieckenst commented:
А и да забыл упомянуть announce - С этой командой ты сможешь послать в первый канал каждого сервера где бот находится сообщение

pieckenst committed:
Поправка локализации

pieckenst committed:
fix

pieckenst committed:
поправки

pieckenst committed:
поправка

pieckenst committed:
en text for openmod

pieckenst committed:
More localize edit

pieckenst committed:
localize prepare

pieckenst committed:
localize

pieckenst committed:
fix localize merge

pieckenst committed:
also fix that

pieckenst committed:
Merge some more

pieckenst committed:
bug fix

Commit:
'Refactored by Sourcery'

pieckenst committed:
Merge pull request #1 from pieckenst/sourcery/master

Sourcery refactored master branch

pieckenst commented:
ждемс ... в принципе ничего не мешает замержить это 
- многофункциональный бот станет

pieckenst committed:
Update admin.py

pieckenst committed:
Update admin.py

pieckenst committed:
Update main.py

arslee07 commented:
> ждемс ... в принципе ничего не мешает замержить это
> 
> * многофункциональный бот станет

пропадёт вся романтика бота - модерация и ничего лишнего. ну вот на кой мне монетка там, или казино?

pieckenst commented:
> > ждемс ... в принципе ничего не мешает замержить это
> > 
> > * многофункциональный бот станет
> 
> пропадёт вся романтика бота - модерация и ничего лишнего. ну вот на кой мне монетка там, или казино?

Могу вырезать монетку и казино

pieckenst committed:
Вырезал монетку, казино,кубик

pieckenst committed:
Delete minigames.py.bak

pieckenst committed:
Delete tools.py.bak

pieckenst committed:
Delete music.py.bak

pieckenst committed:
Delete music.py.bakt

pieckenst committed:
Delete general_commands.py.bak

pieckenst commented:
Я хз чего так надо держаться за только модерацию если больше функционала будет получше

pieckenst committed:
Update commands.json

pieckenst committed:
Update strings.json

pieckenst committed:
Update commands.json

pieckenst committed:
Update strings.json

pieckenst committed:
Update strings.json

pieckenst committed:
Update music.py

arslee07 commented:
не хочу и всё, к тому же чек dev ветку

deepsourcebot committed:
Add .deepsource.toml

deepsource-autofix[bot] committed:
Format code with google-java-format, yapf, isort, rubocop, prettier and black

This commit fixes the style issues introduced in d4c400f according to the output
from google-java-format, yapf, isort, rubocop, prettier and black.

Details: https://deepsource.io/gh/pieckenst/Openmod/transform/21789201-68e4-4c1c-b305-633781c05a96/

deepsource-autofix[bot] committed:
Convert string with anomalous backslash into a raw string

deepsource-autofix[bot] committed:
Format code with google-java-format, yapf, isort, rubocop, prettier and black

deepsource-autofix[bot] committed:
Use identity check for comparison to a singleton

deepsource-autofix[bot] committed:
Format code with google-java-format, yapf, isort, rubocop, prettier and black

deepsource-autofix[bot] committed:
Use identity check for comparison to a singleton

deepsource-autofix[bot] committed:
Format code with google-java-format, yapf, isort, rubocop, prettier and black

pieckenst committed:
Merge pull request #5 from pieckenst/deepsource-fix-b47f8674

Use identity check for comparison to a singleton

pieckenst committed:
Merge pull request #4 from pieckenst/deepsource-fix-91dd4b7c

Use identity check for comparison to a singleton

pieckenst committed:
Merge pull request #3 from pieckenst/deepsource-fix-08a4bb2b

Convert string with anomalous backslash into a raw string

pieckenst committed:
Merge branch 'master' into deepsource-transform-e554704c

pieckenst committed:
Merge pull request #2 from pieckenst/deepsource-transform-e554704c

Format code with google-java-format, yapf, isort, rubocop, prettier and black

pieckenst commented:
> 
> 
> не хочу и всё, к тому же чек dev ветку

Пока что там толком ничего нету ) да и последнюю обнову для python версии можно выкатить просто чтоб было

arslee07 commented:
ну неееееееееее, нинада)

pieckenst commented:
эта версия бота на dart будет готова только к 2024 с таким темпом ) а python версия в 2022 году вообще нахуй сломается со смертью discord.py поэтому замержь и все - разницы не будет

arslee07 commented:
я за гит не шарю, но возможно после мержа будут проблемы

дев ветка зависит от мастера, а значит если я сделаю мерж, то в дев могут появиться конфликты

pieckenst commented:
В dev ветке чисто будет срать что але dev ветка отстает по числу коммитов от master но проблем быть не должно

pieckenst commented:
Всегда можно на основе мастер сделать ветку python и сделать ее основной на какое то время

pieckenst commented:
пока dart версия будет готовиться

arslee07 commented:
ладно, щас посмотрю что да как в пулле и смержу если все ок

arslee07 commented on .deepsource.toml:
```diff
@@ -0,0 +1,46 @@
+  runtime_version = "3.x.x"
+
+[[analyzers]]
+name = "javascript"
```
чё, а причем тут жс?

pieckenst replied:
это фигня для автоматического форматирования кода

pieckenst replied:
ее можно игнорировать) и удалить нахуй

arslee07 commented on src/cogs/announce.py:
```diff
@@ -0,0 +1,69 @@
+
+    @commands.command(description="Global Announcement from bot owner")
+    @commands.is_owner()
+    async def announce(self, ctx: SlashContext, *, content):
```
а вот за это бот отлететь может

pieckenst replied:
Оно заблокированно только на овнера благодаря @commands.is_owner()

pieckenst replied:
кто попало этим не поспамит

pieckenst reviewed (commented):
Вот про причем тут жс

arslee07 commented on src/logs/music.log:
```diff
@@ -0,0 +1,2 @@
+2021-04-28 16:17:13,803:INFO:Tracks added by Aerite#1564 in The hangout
```
bruh

pieckenst replied:
бля логи то там остались) счас почищу

arslee07 commented on src/scripts/desAnime.py:
```diff
@@ -0,0 +1,10 @@
+images = [
```
это че

pieckenst replied:
Это должно хранить ссылки на аниме обои

pieckenst committed:
Delete music.log

arslee07 commented on src/scripts/desNature.py:
```diff
@@ -0,0 +1,39 @@
+images = [
```
bruh, а это

pieckenst replied:
уже убрал же

pieckenst commented on src/scripts/desNature.py:
```diff
@@ -0,0 +1,39 @@
+images = [
```
Тоже ссылки на обои - но на этот раз природы

arslee07 commented on src/scripts/desStarwars.py:
```diff
@@ -0,0 +1,10 @@
+images = [
```
<:OB_think_4:765474279437238292>

pieckenst commented on src/scripts/games.py:
```diff
@@ -0,0 +1,30 @@
+kubik = ["1", "2", "3", "4", "5", "6"]
+
```
Этот скрипт надо выкосить учитывая что я вырезал то что его использует

pieckenst committed:
Delete games.py

arslee07 commented on src/scripts/games.py:
```diff
@@ -0,0 +1,30 @@
+kubik = ["1", "2", "3", "4", "5", "6"]
```
это вероятно тоже не нужно

pieckenst replied:
убрал же

pieckenst committed:
Update music.py

deepsource-autofix[bot] committed:
Format code with black, yapf, isort, prettier, rubocop and google-java-format

This commit fixes the style issues introduced in 80ea0e3 according to the output
from black, yapf, isort, prettier, rubocop and google-java-format.

Details: https://deepsource.io/gh/pieckenst/Openmod/transform/aeea77fa-34a7-4453-9d97-119a0b31f4f0/

arslee07 commented:
в общем, почисти что не нужно, завтра сделаю мерж <:OB_think_4:765474279437238292>

pieckenst committed:
Merge pull request #6 from pieckenst/deepsource-transform-d779d3b2

Format code with black, yapf, isort, prettier, rubocop and google-java-format

pieckenst committed:
Delete music_log.py

pieckenst committed:
add newer translate

pieckenst committed:
add components

deepsource-autofix[bot] committed:
Format code with black, yapf, isort, prettier, rubocop and google-java-format

This commit fixes the style issues introduced in 343be7a according to the output
from black, yapf, isort, prettier, rubocop and google-java-format.

Details: https://deepsource.io/gh/pieckenst/Openmod/transform/ae2c025a-78b2-489a-af3b-9943226266a2/

pieckenst committed:
new help

deepsource-autofix[bot] committed:
Format code with black, yapf, isort, prettier, rubocop and google-java-format

This commit fixes the style issues introduced in 3f70aea according to the output
from black, yapf, isort, prettier, rubocop and google-java-format.

Details: https://deepsource.io/gh/pieckenst/Openmod/transform/91d152b8-7ecc-4191-adfe-46ddc592e729/

pieckenst committed:
Merge pull request #8 from pieckenst/deepsource-transform-cbc1f052

Format code with black, yapf, isort, prettier, rubocop and google-java-format

pieckenst committed:
Merge pull request #7 from pieckenst/deepsource-transform-b14ead39

Format code with black, yapf, isort, prettier, rubocop and google-java-format

pieckenst committed:
Update main.py

pieckenst committed:
edit

pieckenst committed:
edit

pieckenst committed:
edit

pieckenst committed:
Update all cogs

pieckenst commented:
> 
> 
> в общем, почисти что не нужно, завтра сделаю мерж <:OB_think_4:765474279437238292>

уже настало завтра

pieckenst committed:
Update utilities.py

pieckenst committed:
fix broken shit

pieckenst committed:
Update utilities.py

pieckenst committed:
Update utilities.py

pieckenst committed:
Update utilities.py

pieckenst committed:
Update utilities.py

pieckenst committed:
Update utilities.py

pieckenst committed:
Update utilities.py

pieckenst committed:
Update utilities.py

pieckenst committed:
Update utilities.py

pieckenst committed:
Update utilities.py

pieckenst committed:
Update utilities.py

pieckenst committed:
Update music.py

pieckenst committed:
Update music.py

pieckenst commented:
Готово пока что - думаю для мержа норм уже

arslee07 commented on src/cogs/announce.py:
```diff
@@ -0,0 +1,70 @@
+CONFIG = Config()
+
+
+class broadcast(commands.Cog):
```
не совсем понятно зачем это в отдельный класс выносить

pieckenst replied:
чисто потому что хз куда присрать это и чтобы не сломалось что нибудь

arslee07 commented on src/data/settings.json:
```diff
@@ -1 +1,22 @@
-{}
\ No newline at end of file
+{
```
надо бы подчистить

pieckenst replied:
готово

arslee07 commented:
пару комментариев подправь

arslee07 commented:
ну, всмысле что я там написал

pieckenst committed:
Delete .deepsource.toml

pieckenst commented:
убрал deepsource.toml

pieckenst commented:
src/scripts/desAnime, desNature, desStarwars нужны для команды wallpaper и хранят ссылки на обои

pieckenst committed:
remove wallpaper

pieckenst committed:
Delete desAnime.py

pieckenst committed:
Delete desNature.py

pieckenst committed:
Delete desStarwars.py

pieckenst commented:
хотя обои можно и выкосить что я и сделал

pieckenst commented:
announce оставь - никто кроме овнера тобишь тебя не сможет его пользовать

pieckenst commented:
в остальном вроде всё поправил

arslee07 commented:
обрати внимание на файл `src/data/settings.json`
нужен ли он?

pieckenst commented:
Ну как бы бот перестанет отвечать на команды если он полностью пустой
+ у тебя привязка к нему есть в src/cogs/utils.py ( я там толком ничего не трогал кроме форматирования - код твой)

arslee07 commented:
а

pieckenst committed:
fix prefix

pieckenst commented:
поправил еще установку префикса на место

arslee07 commented:
а, лол что

arslee07 commented:
это вот так вот гитхаб работает?

arslee07 commented:
ладно, понатыкай везде типа резолв

pieckenst committed:
clean

pieckenst commented:
вроде бы всё

arslee07 commented:
ок

arslee07 commented:
95 коммитов емаее

pieckenst commented:
нормально работа так да
